pom.xml

```diff
@@ -116,8 +122,8 @@
             <artifactId>easymockclassextension</artifactId>
             <version>2.4</version>
             <scope>test</scope>
-        </dependency>
-        
```

Comment:
whitespace only changes need to be fixed up

Reply:
My text editor has some problems with spaces. Sorry for that.